lib/bemxjst/class-builder.js

```diff
@@ -38,3 +38,27 @@ ClassBuilder.prototype.buildElemClass = function buildElemClass(block,
 ClassBuilder.prototype.split = function split(key) {
   return key.split(this.elemDelim, 2);
 };
+
+/**
+ * Public method available in templates context
+ * @param opts
+ *   @param {String} opts.block
+ *   @param {String} [opts.elem]
+ *   @param {String} [opts.modName]
+ *   @param {String} [opts.modVal]
+ * @example
+ * // returns 'b_m_v'
+ * stringify({ block: 'b', modName: 'm', modVal: 'v' });
+ * @example
+ * // returns 'b__e_m_v'
+ * stringify({ block: 'b', elem: 'e', modName: 'm', modVal: 'v' });
+ * @returns {String} BEM name
+ */
+ClassBuilder.prototype.getEntityName = function getEntityName(opts) {
```

Comment:
2 arguments:
`JSON.stringify` as a widely known example of (abstract) JSON serializer.
`Naming.stringify` as an example of BEMEntity serializer. https://github.com/bem/bem-naming/#stringifyobj